src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw

```diff
@@ -2581,4 +2581,13 @@ Please specify one of them using the --source option to proceed.</value>
   <data name="WINGET_CONFIG_ERROR_UNIT_SETTING_CONFIG_ROOT" xml:space="preserve">
     <value>A unit contains a setting that requires the config root.</value>
   </data>
+  <data name="NoRebootPrivilegeError" xml:space="preserve">
+    <value>The current user does not have the privilege to initiate a reboot.</value>
+  </data>
+  <data name="AllowRebootArgumentDescription" xml:space="preserve">
+    <value>Allows a reboot if applicable</value>
+  </data>
+  <data name="InitiatingReboot" xml:space="preserve">
+    <value>Initiating reboot to complete installation...</value>

```

Comment:
This is not quite true for `APPINSTALLER_CLI_ERROR_INSTALL_REBOOT_REQUIRED_TO_INSTALL` until you hook up `resume`.  I assume that is the plan: `_TO_INSTALL` reboots with a resume but `_TO_FINISH` reboots without a resume?

Side thought, we might already have an issue with `_TO_FINISH` results being treated as an error and thus not recorded properly.  They should probably be treated as a success for post-install purposes.

Reply:
Made changes to ReportInstallerResult so that `APPINSTALLER_CLI_ERROR_INSTALL_REBOOT_REQUIRED_TO_FINISH` is treated as a success and no longer reported as a failed installer result.